cmd/ltctl/reset.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+
+	"github.com/spf13/cobra"
+)
+
+func RunResetCmdF(cmd *cobra.Command, args []string) error {
+	if os.Getenv("SSH_AUTH_SOCK") == "" {
+		return errors.New("ssh agent not running. Please run eval \"$(ssh-agent -s)\" and then ssh-add")
+	}
+
+	config, err := getConfig(cmd)
+	if err != nil {
+		return err
+	}
+
+	output, err := terraform.New(config).Output()
+	if err != nil {
+		return fmt.Errorf("could not parse output: %w", err)
+	}
+
+	if !output.HasAppServers() {
+		return errors.New("no active deployment found")
+	}
+
+	extAgent, err := ssh.NewAgent()
+	if err != nil {
+		return err
+	}
+
+	appClient, err := extAgent.NewClient(output.Instances.Value[0].PublicIP)
+	if err != nil {
+		return fmt.Errorf("error in getting ssh connection %w", err)
+	}
+
+	agentClient, err := extAgent.NewClient(output.Agents.Value[0].PublicIP)
+	if err != nil {
+		return fmt.Errorf("error in getting ssh connection %w", err)
+	}
+
+	fmt.Println("Are you sure you want to delete everything? All data will be permanently deleted? [y/N]")
+	var confirm string
+	fmt.Scanln(&confirm)
+	if !regexp.MustCompile(`(?i)^(y|yes){1}?$`).MatchString(confirm) {
+		return nil
```

Comment:
We may want to return err here like we did [here](https://github.com/mattermost/mattermost-load-test-ng/blob/f02dfcc0a29f85cd0a3c6d697c2b6a896091466f/cmd/ltctl/report.go#L62). Or vice versa. 0/5 on returning error though. For instance `rm -i file` returns `0` when entered non-yes input.

Reply:
My idea is that if the answer is not yes the program should exit without errors (code 0). It's not an error to cancel the operation in my opinion.

Reply:
I see, I just wanted to point out we return errors in the other case. This is good to go as-is.

Reply:
Yes, I can update the other case for consistency :+1: